crashsimilarity/stacktrace_utils.py

```diff
@@ -0,0 +1,52 @@
+import json
```

Comment:
Could you rename this file `stacktrace.py` directly?

Reply:
Fixed @marco-c